sphinx_search/_static/js/rtd_sphinx_search.js

```diff
@@ -406,19 +418,74 @@ const removeSearchModal = () => {
     search_outer_wrapper.classList.remove("display-block");
 };
 
+/**
+ * Returns the search URL
+ *
+ * @param {Object} searchParams params for search
+ */
+const getSearchUrl = searchParams => {
+    const api_host = getProjectInfo().api_host;
+    const url =
+        api_host + "/api/v2/docsearch/?" + convertObjToUrlParams(searchParams);
+    return url;
+};
+
+/**
+ * Returns project info
+ */
+const getProjectInfo = () => {
+    return {
+        project: READTHEDOCS_DATA.project,
+        version: READTHEDOCS_DATA.version,
+        language: READTHEDOCS_DATA.language || "en",
+        api_host: READTHEDOCS_DATA.api_host
+    };
+};
+
+/**
+ * Returns the URL parameters in the form of Object
+ *
+ * @return {Object} url params
+ */
+const getUrlVars = () => {
+    let vars = {};
+    let params = window.location.href.replace(
+        /[?&]+([^=&]+)=([^&]*)/gi,
+        (m, key, value) => {
+            vars[key] = value;
+        }
+    );
+    return vars;
+};
+
+/**
+ * Copy the text passes as argument.
+ *
+ * @param {String} str string to be copied to clipboard
+ */
+const copyToClipBoard = str => {
+    let textarea = createDomNode("textarea", {
+        readonly: "",
+        style: "position: absolute; left: -9999px;"
+    });
+    textarea.value = str;
+    document.body.appendChild(textarea);
+    textarea.select();
+    document.execCommand("copy");
+    document.body.removeChild(textarea);
+};
```

Comment:
Same here, is this really not a standard thing?

Reply:
No.
here also -- no utility of jquery or javascript is there which dirctly copies the text.